### PR TITLE
[ERSSUP-55732]-[]-[Validate AAL3 Security Level]-[RG]

### PR DESCRIPTION
**Pull request by RajaGovindharaj16**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/policies/OAuthV2.VerifyAccessToken.xml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <OAuthV2 async="false" continueOnError="false" enabled="true" name="OauthV2.VerifyAccessToken">
   <Operation>VerifyAccessToken</Operation>
+  <Scope>urn:nhsd:apim:app:level3:e-referrals-service-api urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api</Scope>
```
Comment (nhsd-jack-wainwright) [inline proxies/live/apiproxy/policies/OAuthV2.VerifyAccessToken.xml:3]: Does this not allow for tokens created via the application-restricted route to be accepted as well as user restricted? I think only the "urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api" scope should be being included here.

File: proxies/sandbox/apiproxy/policies/OAuthV2.VerifyAccessToken.xml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <OAuthV2 async="false" continueOnError="false" enabled="false" name="OauthV2.VerifyAccessToken">
   <Operation>VerifyAccessToken</Operation>
+  <Scope>urn:nhsd:apim:app:level3:e-referrals-service-api urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api</Scope>
```
Comment (nhsd-jack-wainwright) [inline proxies/sandbox/apiproxy/policies/OAuthV2.VerifyAccessToken.xml:3]: As above I think only the AAL3 scope should be being included here as we don't yet want to support application-restricted access.

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Should we also be removing the "'urn:nhsd:apim:app:level3:e-referrals-service-api'" scope from our manifest_template.yml file.

**Review by nhsd-hubert-raich — DISMISSED** [review]

**Pull request closed without merging**